utilities/src/main/java/io/syndesis/qe/bdd/validation/SfValidationSteps.java

```diff
@@ -174,12 +174,18 @@ public void validateIntegration(String twAccount, String record) {
         log.info("Salesforce contains contact with T integration test finished.");
     }
 
-    @Then("check SF does not contain contact for tw accound: \"([^\"]*)\"")
+    @Then("check SF does not contain contact for tw account: \"([^\"]*)\"")
     public void checkSfDoesNotContain(String twAccount) {
         assertThat(getSalesforceContact(salesforce, accountsDirectory.getAccount(twAccount)
                 .get().getProperty("screenName")).isPresent()).isEqualTo(false);
     }
 
+    @Then("check SF contains contact for tw account: \"([^\"]*)\"")
+    public void checkSfDoesContain(String twAccount) {
+        assertThat(getSalesforceContact(salesforce, accountsDirectory.getAccount(twAccount)
```

Comment:
Wouldn't be a private method with boolean argument be better than code replication?

Reply:
yes, it would be better